gradle.properties

```diff
@@ -1,2 +1,3 @@
 kotlinVersion=1.3.70
 kotlin.code.style=official
+systemProp.org.gradle.internal.publish.checksums.insecure=true
```

Comment:
This doesn't seem so cool, but looks like is the suggested thing to do 🤔

Reply:
i know but all the security checks pass on maven central, so is actually secure 😄